lib/wallet/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -491,6 +491,8 @@ import Test.QuickCheck.Extra
     ( reasonablySized )
 import Test.QuickCheck.Gen
     ( sublistOf )
+import Test.QuickCheck.Instances
+    ()
```

Comment:
Oh dear, we already depend on the `quickcheck-instances` package, in the `library` section of `cardano-wallet` no less. 😱

https://github.com/input-output-hk/cardano-wallet/blob/29b58ca59c8bde9083ea2bafe756b9797423744e/lib/wallet/cardano-wallet.cabal#L156

It's probably out of scope for this pull request — but could you make a ticket for removing our dependency on `quickcheck-instances`? We do have a [coding standard](https://input-output-hk.github.io/adrestia/code/Coding-Standards#test-files-are-separated-and-self-contained) on that, after all:

> Arbitrary instances are not shared across test files and are defined locally

Reply:
Done